### ISSUE TITLE
Startup not triggered with multiple build inputs

### ISSUE THREAD

**Issue by dcarr45** [electron-vite/vite-plugin-electron#182]
I may be misunderstanding the expected config, but this configuration of multiple input files that was working in 0.14.1 no longer works in 0.15.x due to #166:
```
{
    entry: ['main.ts', 'preload.ts', 'redis.ts'],
    vite: {
      build: {
        outDir: './dist/electron',
        emptyOutDir: true,
        rollupOptions: {
          output: {
            format: 'commonjs',
          },
        },
        sourcemap: 'inline',
      },
    },
    onstart(args) {
      void args.startup(['./dist/electron/main.js']);
    },
  },
```
It seems that the `closeBundle()` event is only fired once during `vite serve`, regardless of the number of entry/input files, so this condition
```
if (++closeBundleCount < entryCount) return
```
is never met.

I'm on `vite@4.5.0`

**Comment by caoxiemeihao** [comment]
Hey! thanks for the tip!

**Comment by GeoffreyChen777** [comment]
Hi, I met the same issue.

Finally I found that, in v0.15.x, we should config the plugin like:

```
 electron([
        {
          entry: "app/main/main-entry.ts",
          vite: {
            
          }
        },
        {
          entry: "app/main/second-entry.ts",
          vite: {
            
          }
        }
]),

```

**Comment by caoxiemeihao** [comment]
<img width="877" alt="image" src="https://github.com/electron-vite/vite-plugin-electron/assets/26263658/6b745341-ad47-46ae-aaa4-28ed12372dd1">

@GeoffreyChen777 Hey! 👋
This BUG fixed in `v0.15.5`, doesn't this work for you?

See here 👉 https://github.com/electron-vite/vite-plugin-electron/commit/f57a60a1a6209c934564b23c66a03485b4e2d74c
